ocrd_utils/ocrd_utils/constants.py

```diff
@@ -1,10 +1,13 @@
 """
 Constants for ocrd_utils.
 """
-from pkg_resources import get_distribution
 from re import compile as regex_compile
 from os import environ
 from os.path import join, expanduser
+try:
```

Comment:
Yes, good idea. It didn't work at first at installation time, but that was because I listed the `importlib_*` requirements in `ocrd/requirements.txt` instead of `ocrd_utils/requirements.txt`